ios/BlurView.m

```diff
@@ -37,12 +37,13 @@ - (void)setBlurType:(NSString *)blurType
     } else {
         self.blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleDark];
```

Comment:
It seems like these methods just need to be reverted to using `[BlurAmount +effectWithStyle]` instead of `UIBlurEffect`